app/views/layouts/application.html.erb

```diff
@@ -16,6 +16,7 @@
     <p class="notice"><%= notice %></p>
     <p class="alert"><%= alert %></p>
```

Comment:
consider taking these out if we're rendering on line 19

Reply:
All done captain!

Reply:
Aye Aye!